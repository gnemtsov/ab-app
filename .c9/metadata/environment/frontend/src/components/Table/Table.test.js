{"filter":false,"title":"Table.test.js","tooltip":"/frontend/src/components/Table/Table.test.js","undoManager":{"mark":-1,"position":-1,"stack":[]},"ace":{"folds":[],"scrolltop":9860,"scrollleft":0,"selection":{"start":{"row":0,"column":0},"end":{"row":0,"column":0},"isBackwards":false},"options":{"guessTabSize":true,"useWrapMode":true,"wrapToView":true},"firstLineState":{"row":292,"state":"start","mode":"ace/mode/javascript"}},"timestamp":1541956724789,"hash":"40a0816af479c43978bab5c749f5adcd401b7a5f"}