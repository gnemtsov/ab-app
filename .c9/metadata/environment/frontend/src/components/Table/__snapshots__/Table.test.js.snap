{"filter":false,"title":"Table.test.js.snap","tooltip":"/frontend/src/components/Table/__snapshots__/Table.test.js.snap","undoManager":{"mark":-1,"position":-1,"stack":[]},"ace":{"folds":[],"scrolltop":0,"scrollleft":0,"selection":{"start":{"row":0,"column":0},"end":{"row":0,"column":0},"isBackwards":false},"options":{"guessTabSize":true,"useWrapMode":true,"wrapToView":true},"firstLineState":0},"timestamp":1541956741545,"hash":"0ad36379539dcabed179a69155040eeae40bda87"}